game/scripts/vscripts/Sections/Chapter2/SectionTower.ts

```diff
@@ -60,7 +60,7 @@ const onStart = (complete: () => void) => {
     const items: CDOTA_Item[] = []
 
     // Get or create the Dire T1 tower
-    const direTopTower = getDireTopTower();
+    const direTopTower = getOrError(getDireTopTower());
```

Comment:
This will fail if the dire tower is dead right? `getDireTopTower()` will return undefined and getOrError will error

Reply:
Yeah that (should) works when you skip back and forth between sections that have/don't have towers. Haven't got any errors.